src/renderers/shared/reconciler/__tests__/ReactIdentity-test.js

```diff
@@ -107,12 +107,12 @@ describe('ReactIdentity', function() {
     var Wrapper = React.createClass({
 
       render: function() {
-        var span1 = <span ref="span1" key={key} />;
-        var span2 = <span ref="span2" />;
+        var s1 = <span ref="span1" key={key} />;
```

Comment:
Yeah, I think most of the `no-shadow` rules triggered in places where the variable was defined below the shadowing scope because they probably would've been caught by a human earlier otherwise.